andela/ah-frontend-targaryen
# #161820823 Users should be able to edit their profiles

#### What does this PR do?
This PR enables logged in users to edit their profiles.
#### How should this be manually tested?
Run the Authors Heaven application `npm start`
Navigate to the login page and login using your account details.
On successful login, there should be a redirection to the dashboard page
Click on the user icon and select profile
On the profiles page, click edit

#### What are the relevant pivotal tracker stories?

[#161820823](https://www.pivotaltracker.com/story/show/161820823)

#### Any background context you want to add?
N/A

## Checklist: I completed these to help reviewers
